### PR TITLE
Feature/fix box contains

### PR DESCRIPTION
**Pull request by jvbsl**
* Use latest C# version(necessary for me with dotnet 3.0 preview, doesn't seem to recognize 7.3)
* Make contains boundary exclusive, as tests do test for that -> fixed tests by doing that
* Make contains overload with bool parameter to test boundary inclusive as well
* Use boundary inclusive contains for inflate tests

### CODE REVIEW

File: tests/OpenToolkit.Tests/Box3Tests.fs [hunk 1]
```diff
@@ -155,10 +155,10 @@ module Box3 =
         
         [<Property>]
         let ``After inflating the point should be enclosed in the box`` (b1 : Box3, v1 : Vector3) =
-            Assert.True(b1.Inflated(v1).Contains(v1))
+            Assert.True(b1.Inflated(v1).Contains(v1, true))
 
         [<Property>]
-        let ``Box2.Inflate is equivelant to Box2.Inflated`` (b1 : Box3, v1 : Vector3) =
+        let ``Box2.Inflate is equivalent to Box2.Inflated`` (b1 : Box3, v1 : Vector3) =
```
Comment (realvictorprm) [inline tests/OpenToolkit.Tests/Box3Tests.fs]: lol that change, hilarious. Good catch! 🥇

**Comment by varon** [comment]
We've got a little code duplication here; the overload should probably defer to the other method.

I'll merge for now as this fixes our tests!

**Comment by varon** [comment]
Thanks a ton @jvbsl !

**Comment by jvbsl** [comment]
the code duplication was on purpose for now, for performance, and I didn't want to AggressiveInline everything :D

**Comment by varon** [comment]
Figures. Hence the merge.

**Pull request merged**